### PR TITLE
docs(changelog.md): update for release 0.30.0

### PR DESCRIPTION
**Pull request by araujof**
Signed-off-by: Frederico Araujo <frederico.araujo@ibm.com>

**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

NONE

**What this PR does / why we need it**:

Additions to the CHANGELOG for Falco 0.30.0

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:

/milestone 0.30.0

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+* new(outputs): expose rule tags and event source in gRPC and json outputs [[#1714](https://github.com/falcosecurity/falco/pull/1714)] - [@jasondellaluce](https://github.com/jasondellaluce)
+* new(userspace/falco): add customizable metadata fetching params [[#1667](https://github.com/falcosecurity/falco/pull/1667)] - [@zuc](https://github.com/zuc)
+
+
+### Minor Changes
+
+* update: bump driver version to 3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4 [[#1744](https://github.com/falcosecurity/falco/pull/1744)] - [@zuc](https://github.com/zuc)
+* docs: clarify that previous Falco drivers will remain available at https://download.falco.org and no automated cleanup is run anymore [[#1738](https://github.com/falcosecurity/falco/pull/1738)] - [@leodido](https://github.com/leodido)
+* update(outputs): add configuration option for tags in json outputs [[#1733](https://github.com/falcosecurity/falco/pull/1733)] - [@jasondellaluce](https://github.com/jasondellaluce)
+* update: driver version is 5727c456ce22f3c1da8c3b1d7d6b6937a9b2126b now [[#1734](https://github.com/falcosecurity/falco/pull/1734)] - [@zuc](https://github.com/zuc)
```
Comment (leodido) [inline CHANGELOG.md:20]: The double driver version update can be very confusing for users and brings no valuable info to them.

@leogr I suggest to go into this PR and insert NONE in the release-block portion of the body 


Then, this change log would need to be regenerated and (force) pushed on this branch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+* build: remove unused `ncurses` dependency [[#1658](https://github.com/falcosecurity/falco/pull/1658)] - [@leogr](https://github.com/leogr)
+* build(.circleci): use new Debian 11 package names for python-pip [[#1712](https://github.com/falcosecurity/falco/pull/1712)] - [@zuc](https://github.com/zuc)
+* build(docker): adding libssl-dev, upstream image reference pinned to `debian:buster` [[#1719](https://github.com/falcosecurity/falco/pull/1719)] - [@michalschott](https://github.com/michalschott)
+* fix(test): avoid output_strictly_contains failures [[#1724](https://github.com/falcosecurity/falco/pull/1724)] - [@jasondellaluce](https://github.com/jasondellaluce)
+* Remove duplicate allowed ecr registry rule [[#1725](https://github.com/falcosecurity/falco/pull/1725)] - [@TomKeyte](https://github.com/TomKeyte)
+* docs(RELEASE.md): switch to 3 releases per year [[#1711](https://github.com/falcosecurity/falco/pull/1711)] - [@leogr](https://github.com/leogr)
+* docs: CHANGELOG for 0.29.1 cleanup [[#1688](https://github.com/falcosecurity/falco/pull/1688)] - [@leodido](https://github.com/leodido)
```
Comment (leodido) [inline CHANGELOG.md:45]: Same as the other comment above IMO

**Review by leodido — COMMENTED** [review]
Please consider my comments

File: CHANGELOG.md (hunk 1 above)
Comment (leogr) [reply CHANGELOG.md:20]: @leodido Done! And thank you for having caught that. 

@araujof could we regenerate the changelong?

File: CHANGELOG.md (hunk 2 above)
Comment (leogr) [reply CHANGELOG.md:45]: The release note of #1688 is already set to `NONE`, however, the tool uses the PR title for the `Non user-facing changes`.
So I believe we have just to manually remove this line.

**Comment by leogr** [comment]
Hey @araujof 

I addressed the first suggestion from @leodido. The other suggestion needs to be manually applied, IMO.

File: CHANGELOG.md (hunk 1 above)
Comment (araujof) [reply CHANGELOG.md:20]: @leogr will do! Thanks!

**Comment by araujof** [comment]
@leogr @leodido I regenerated the CHANGELOG, manually removed 1744 and 1688, amended and force-pushed my previous commit. Thanks for your help!

**Review by leodido — APPROVED** [review]

**Review by leogr — APPROVED** [review]
Thank you!

**Pull request merged**